frontend/src/components/DecayInformation.vue

```diff
@@ -68,24 +68,24 @@ export default {
   props: {
     decay: {
       balance: '',
-      decay_duration: '',
-      decay_start: 0,
-      decay_end: 0,
-      decay_start_block: 0,
+      decayDuration: '',
+      decayStart: 0,
+      decayEnd: 0,
+      decayStartBlock: 0,
     },
     decaytyp: { type: String, default: '' },
   },
   computed: {
     decayStartBlockTextShort() {
-      return this.decay.decay_start_block
-        ? this.$t('decay.decayStart') + this.$d(this.$moment.unix(this.decay.decay_start_block))
+      return this.decay.decayStartBlock
+        ? this.$t('decay.decayStart') + this.$d(this.$moment.unix(this.decay.decayStartBlock))
```

Comment:
Why does this not arrive as date but int?

Reply:
I think it is the number of milliseconds since 1970